Automattic/wp-calypso
# [Domains] Split out Kracken feature flags

This change feature gates more specific portions of the Kracken redesign: filtering and pagination. It also disables filtering and pagination in our staging environment in preparation for Kracken's internal call-for-testing.

### Testing instructions
0. WIP

1. Spin up this branch locally in development.
2. Navigate to [`/start/domains`](http://calypso.localhost:3000/start/domains).
3. Ensure that filter and pagination controls are present.
4. Spin up this branch in a `stage` environment, like so: `npm run build-docker && `.
5. Ensure that [`/start/domains`](http://calypso.localhost:3000/start/domains) has neither filter controls nor pagination controls.
